src/commands/Movie.js

```diff
@@ -0,0 +1,69 @@
+const { Command } = require('@dyno.gg/dyno-core');
+
+class Movie extends Command {
+	constructor(...args) {
+		super(...args);
+
+        this.aliases        = ['movie'];
+        this.module         = 'Fun';
+        this.description    = 'Get info on a movie.';
+        this.usage          = 'movie [movie name]';
+        this.example        = 'movie Deadpool';
+        this.cooldown       = 3000;
+        this.expectedArgs   = 1;
+	}
+
+	async execute({ message, args }) {
+		const MovieDB = require('movieDB')(this.config.movieDBKey);
```

Comment:
Should be at the top of the file, or we'll be requiring it every time the command is run, can instantiate it in the constructor as a property prefixed by `_`
```
const movieDB = require('movieDB');
...
constructor() {
  this._movieDB = movieDB(this.config.movieDBKey);
}
```